x-pack/plugin/esql/qa/server/mixed-cluster/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/mixed/MixedClusterEsqlSpecIT.java

```diff
@@ -74,14 +68,6 @@ public MixedClusterEsqlSpecIT(
     protected void shouldSkipTest(String testName) throws IOException {
         super.shouldSkipTest(testName);
         assumeTrue("Test " + testName + " is skipped on " + bwcVersion, isEnabled(testName, instructions, bwcVersion));
-        if (mode == ASYNC) {
-            assumeTrue("Async is not supported on " + bwcVersion, supportsAsync());
```

Comment:
Given that 9.0 will always be talking to a node that supports features, I've just removed this check completely

Reply:
++